ngraph/test/type_prop/roll.cpp

```diff
@@ -0,0 +1,139 @@
+//*****************************************************************************
+// Copyright 2017-2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "gtest/gtest.h"
+#include "ngraph/ngraph.hpp"
+#include "util/type_prop.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+TEST(type_prop, roll_output_shape_type_test)
+{
+    auto arg = make_shared<op::Parameter>(element::f32, Shape{3, 3, 4, 1, 5});
```

Comment:
Do not use `op::OpName` approach. Instead use `opsetX::OpName` approach. In this case use `opset7` everywhere. Please, fix all test cases.